src/modules/position-creator/services/position.creator.transaction.ts

```diff
@@ -168,15 +200,15 @@ export class PositionCreatorTransactionService {
         const endpointArgs = lockEpochs
```

Comment:
the ternary operator is recommended to be used only on one liners for readability

Reply:
done